src/app/public/modules/auth-http-client/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './auth-options';
 export * from './auth-http-client.module';
+export { SKY_AUTH_DEFAULT_PERMISSION_SCOPE } from './auth-interceptor-default-permission-scope';
```

Comment:
Can be `export *` since it's in its own file now.

Reply:
Fixed.